ctrd/image.go

```diff
@@ -38,16 +38,17 @@ func (c *Client) ListImages(ctx context.Context, filter ...string) ([]types.Imag
 	}
 
 	images := make([]types.Image, 0, 32)
-	digestPrefix := "sha256:"
 	for _, image := range imageList {
 		descriptor := image.Target()
-		digest := []byte(descriptor.Digest)
+		digest := descriptor.Digest
 		size := descriptor.Size
 
+		// TODO(Wei Fu): the go-digest will deprecate the Hex() method.
+		// We need to use dig.Encoded() if update the go-digest version.
```

Comment:
/dig.Encoded/digest.Encoded/ ???

Reply:
Sorry about that. I remove the dig. since the digest is used as alias for go-digest